recipe/meta.yaml

```diff
@@ -7,12 +7,14 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - scattering.patch
-    - boostpythonsuffixes.patch
+# fn: {{ name }}-{{ version }}.tar.gz
+# url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+# sha256: {{ sha256 }}
+# patches:
+#   - scattering.patch
+#   - boostpythonsuffixes.patch
+  git_url: https://github.com/pavoljuhas/pyobjcryst.git
```

Comment:
Please build from tarballs https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
Will do when the build passes.  Note the commit message starts with "DROPME".